gradle-server/build.gradle

```diff
@@ -7,11 +7,12 @@ plugins {
 description = 'vscode-gradle :: gradle-server'
 
 java {
-  sourceCompatibility = JavaVersion.VERSION_1_8
-  targetCompatibility = JavaVersion.VERSION_1_8
+  sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
will it still work for java 8 developers?

Reply:
if we change this `sourceCompatibility` to Java 11, then this will not support Java 8 users.